app/_hub/kong-inc/kong-terraform-aws/_index.md

```diff
@@ -13,6 +13,7 @@ license_type: Apache-2.0
 kong_version_compatibility:
```

Comment:
This page doesn't have a plugin version or Gateway version dropdown in the netlify preview, not sure if that's intended or not.

Reply:
Yep, that's intended. If it's a `type: integration` plugin, they never have versions. This is one of the only (maybe the only) plugin like this that's managed by Kong; the rest are all external.